### PR TITLE
Compression options for AlignAndFocusPowderFromFiles

### PR DESCRIPTION
**Pull request by dmitry-ganyushin**
[6060: Add load w/ compression parameters to AlignAndFocusPowderFromFiles](https://ornlrse.clm.ibmcloud.com/ccm/resource/itemName/com.ibm.team.workitem.WorkItem/6060)
### Description of work
Added an additional parameter  AlignAndFocusPowderFromFiles to determine whether its call to LoadEventNexus should enable compressing events during load. The parameter is called "MinSizeCompressOnLoad" which is a floating point number of Gbytes. 
.

### CODE REVIEW

**Review by peterfpeterson — CHANGES_REQUESTED** [review]
This missed the mark. The request was "AlignAndFocusPowderFromFiles needs additional parameter(s) to determine whether its call to LoadEventNexus should enable compressing events during load." When it is determined that the file should be compressed on load, the parameters `CompressTolerance` and `CompressBinningMode` need to be supplied to `LoadEventNexus`. The `__compressEvents()` method is called by the `__accummlate()` method which is run after a chunk is processed. These methods should not be modified.

This is also missing release notes.

File: Framework/PythonInterface/plugins/algorithms/AlignAndFocusPowderFromFiles.py [hunk 1]
```diff
@@ -213,6 +222,11 @@ def __createLoader(self, filename, wkspname, progstart=None, progstop=None, skip
         loader.setPropertyValue("Filename", filename)
         loader.setPropertyValue("OutputWorkspace", wkspname)
 
+        if self.do_compression:
+            self.kwargs = self.__getAlignAndFocusArgs()
+            loader.setPropertyValue("CompressTolerance", str(self.kwargs["CompressTolerance"]))
```
Comment (peterfpeterson) [inline Framework/PythonInterface/plugins/algorithms/AlignAndFocusPowderFromFiles.py:227]:
```suggestion
            loader.setProperty("CompressTolerance", float(self.kwargs["CompressTolerance"]))
```

Comment (dmitry-ganyushin) [reply Framework/PythonInterface/plugins/algorithms/AlignAndFocusPowderFromFiles.py]: Modified as suggested

**Review by peterfpeterson — APPROVED** [review]
I tried this in three modes:
* without the `CompressTolerance` parameter - did not compress on load
* with the `CompressTolerance` > filesize - did not compress on load
* with the `CompressTolerance` < filesize - did compress on load

**Pull request merged**